packages/angular-cli/models/webpack-build-development.ts

```diff
@@ -1,14 +1,3 @@
-const ExtractTextPlugin = require('extract-text-webpack-plugin');
-
 export const getWebpackDevConfigPartial = function(projectRoot: string, appConfig: any) {
-  return {
-    output: {
-      filename: '[name].bundle.js',
-      sourceMapFilename: '[name].bundle.map',
-      chunkFilename: '[id].chunk.js'
-    },
-    plugins: [
-      new ExtractTextPlugin({filename: '[name].bundle.css'})
-    ]
-  };
+  return { };
```

Comment:
That's the reason I was going to add it. But it also stabilizes the module id's between builds which has benefits for cache busting.  The hash won't change due to webpack module id renumbering.  Nothing bad happens without it; the hash just might change more often than needed.  And this isn't really relevant for HMR and probably not for dev builds.
For production builds there is `HashedModuleIdsPlugin`, which hashes the names to obfuscate the file path details.